geopmdpy/geopmdpy/session.py

```diff
@@ -14,8 +14,13 @@
 from . import topo
 from . import pio
 from . import loop
+from . import stats
 from . import __version_str__
 
+try:
+    from mpi4py import MPI
```

Comment:
Add to requirements.txt?

Reply:
Since it is optional and also has an unspecified dependency on some flavor of MPI development packages be installed, I've opted to leave it out.

Reply:
You could add an optional dependency if you think there's value in one-liner pip installs.

E.g., `pip install geopmdpy[mpi]` could install `geopmdpy` as normal, but also add `mpi4py`.

https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#optional-dependencies

If you do decide to try that out, try it on python3.6 first in case it's another one of those packaging things that doesn't work well until we drop 3.6 support.